### PR TITLE
Add Transaltion section README.md

### PR DESCRIPTION
**Pull request by jannesblobel**
AppFlowy has a lot of missing translations. 
I thought you could use your community for the missing translation.

### Feature Preview

---

#### PR Checklist

- [ ] My code adheres to the [AppFlowy Style Guide](https://appflowy.gitbook.io/docs/essential-documentation/contribute-to-appflowy/software-contributions/submitting-code/style-guides)
- [ ] I've listed at least one issue that this PR fixes in the description above.
- [ ] I've added a test(s) to validate changes in this PR, or this PR only contains semantic changes.
- [ ] All existing tests are passing.

I added the inlang Badge to the readme. 
that makes it easier for the community to see how much of the app needs to be translated. The idea behind the badge is to leverage the power of the community :) 

### Badge Preview 
For AppFlowy
[translation badge](https://inlang.com/editor/github.com/AppFlowy-IO/AppFlowy?ref=badge)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,14 @@ Contributions make the open-source community a fantastic place to learn, inspire
 If your Pull Request is accepted as it fixes a bug, adds functionality, or makes AppFlowy's codebase significantly easier to use or understand, **Congratulations!** If your administrative and managerial work behind the scenes sustains the community, **Congratulations!** You are now an official contributor to AppFlowy. Get in touch with us ([link](https://tally.so/r/mKP5z3)) to receive the very special Contributor T-shirt!
 Proudly wear your T-shirt and show it to us by tagging [@appflowy](https://twitter.com/appflowy) on Twitter.
 
+
+## Translations 🌎🗺
+
+[![translation badge](https://inlang.com/badge?url=github.com/AppFlowy-IO/AppFlowy)](https://inlang.com/editor/github.com/AppFlowy-IO/AppFlowy?ref=badge)
+
+To add translations, you can manually edit the JSON translation files in /frontend/appflowy_flutter/assets/translations, use the [inlang online editor](http://localhost:3000/editor/github.com/AppFlowy-IO/AppFlowy), or run npm machine-translate to add missing translations
```
Comment (LucasXu0) [inline README.md:76]: Is it a typo? why using localhost here?

Comment (bijaykumarpun) [inline README.md:76]: Can use backticks for code ; `npm run machine-translate`.
Even for `/frontend/appflowy_flutter/assets/translations` for readability.

File: README.md [hunk 2]
```diff
@@ -68,6 +68,15 @@ Contributions make the open-source community a fantastic place to learn, inspire
 If your Pull Request is accepted as it fixes a bug, adds functionality, or makes AppFlowy's codebase significantly easier to use or understand, **Congratulations!** If your administrative and managerial work behind the scenes sustains the community, **Congratulations!** You are now an official contributor to AppFlowy. Get in touch with us ([link](https://tally.so/r/mKP5z3)) to receive the very special Contributor T-shirt!
 Proudly wear your T-shirt and show it to us by tagging [@appflowy](https://twitter.com/appflowy) on Twitter.
 
+
+## Translations 🌎🗺
+
+[![translation badge](https://inlang.com/badge?url=github.com/AppFlowy-IO/AppFlowy)](https://inlang.com/editor/github.com/AppFlowy-IO/AppFlowy?ref=badge)
+
+To add translations, you can manually edit the JSON translation files in `/frontend/appflowy_flutter/assets/translations`, use the [inlang online editor](https://inlang.com/editor/github.com/AppFlowy-IO/AppFlowy), or run `npx inlang machine translate`
+ to add missing translations
```
Comment (Xazin) [inline README.md:77]:
```suggestion
 to add missing translations.
```

Comment (Xazin) [reply README.md:77]: Does this need to be on a newline?

Comment (jannesblobel) [reply README.md]: I guess I made every mistake you can make. 
It's fixed

**Review by Xazin — APPROVED** [review]

**Comment by Xazin** [comment]
CC @annieappflowy

**Pull request merged**